### PR TITLE
Add pull statistics in the user's profile page

### PR DESCRIPTION
**Pull request by xel1045**
# Description

The PR add "Assigned pulls" and "Authored pulls" to a user's profile page.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

This PR was manually tested.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -75,6 +75,8 @@ en:
   label_add_more_commits: "Add more commits by pushing to the <code class='pull__branch'>%{branch}</code> branch."
   label_merge_instructions_title: Merging via command line
   label_merge_instructions_description: If you do not want to use the merge button or an automatic merge cannot be performed, you can perform a manual merge on the command line.
+  label_assigned_pulls: Assigned pulls
```
Comment (martinblanchette) [inline config/locales/en.yml]: You can pluralize your locale, with something like that :

```
<%= link_to l(:label_assigned_pulls, count: Pull.open.assigned_to(@user).count), pulls_path(:set_filter => 1, :assigned_to_id => ([@user.id] + @user.group_ids).join("|"), :sort => 'updated_on:desc') %>
```

```
label_assigned_pulls:
    one: Assigned pull (1)
    other: Assigned pulls (%{count})
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -75,6 +75,8 @@ en:
...
+  label_authored_pulls: Authored pulls
```
Comment (martinblanchette) [inline config/locales/en.yml]: You can pluralize your locale, with something like that :

```
<%= link_to l(:label_authored_pulls, count: Pull.where(:author_id => @user.id).count), pulls_path(:set_filter => 1, :assigned_to_id => ([@user.id] + @user.group_ids).join("|"), :sort => 'updated_on:desc') %>
```

```
label_authored_pulls:
    one: Authored pull (1)
    other: Authored pulls (%{count})
```

**Review by martinblanchette — COMMENTED** [review]
This is pretty good 👌. However, I made two suggestions 😬.

**Comment by xel1045** [comment]
Screen Shot 2019-09-06 at 10 58 39

I'm not sure what is better. Apply you suggestion or be consistent with Redmine.

What do you think?

**Comment by martinblanchette** [comment]
Good catch! In this let stick with Redmine conventions.

**Review by martinblanchette — APPROVED** [review]
👍

**Comment by xel1045** [comment]
OK, thanks for your advise. I'll merge this that way.

**Pull request merged**